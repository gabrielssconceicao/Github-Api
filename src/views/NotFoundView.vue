<script setup lang="ts"></script>
<template>
  <section class="w-screen h-dvh flex flex-col gap-4 justify-center items-center">
    <h2 class="text-3xl md:text-5xl font-bold text-white">Not Found</h2>
    <RouterLink to="/" class="text-white font-bold border-2 border-white p-2 text-base md:text-3xl">Go Back</RouterLink>
  </section>
</template>
<style scoped>
h2 {
  text-shadow: 2px 1px 2px #ff0000, 2px -1px 2px #0000ff;
}
</style>
