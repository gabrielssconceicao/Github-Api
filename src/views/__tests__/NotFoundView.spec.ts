import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';
import NotFoundView from '../NotFoundView.vue';

describe('<NotFoundView />', () => {
  it('should render correctly', () => {
    const wrapper = mount(NotFoundView);
    expect(wrapper.exists()).toBe(true);
  });
  it('should match snapshot', () => {
    const wrapper = mount(NotFoundView);
    expect(wrapper.element).toMatchSnapshot();
  });
});
