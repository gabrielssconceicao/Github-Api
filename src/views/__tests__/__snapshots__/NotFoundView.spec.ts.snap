// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<NotFoundView /> > should match snapshot 1`] = `
<section
  class="w-screen h-dvh flex flex-col gap-4 justify-center items-center"
  data-v-071430af=""
>
  <h2
    class="text-3xl md:text-5xl font-bold text-white"
    data-v-071430af=""
  >
    Not Found
  </h2>
  <routerlink
    class="text-white font-bold border-2 border-white p-2 text-base md:text-3xl"
    data-v-071430af=""
    to="/"
  >
    Go Back
  </routerlink>
</section>
`;
